src/Analytics/UAExperimentData.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace PhpAb\Analytics;
+
+use Webmozart\Assert\Assert;
+
+class UAExperimentData
```

Comment:
I don't know how I would actually index them:
You would have these cases.

- PhpAb\Analytics\Google\DataCollector
- PhpAb\Analytics\Renderer\GoogleUAnalytics

Which would mix at the same level "vendor" (google) and "function" (Renderer).
Another option would be:
- PhpAb\Analytics\Google\DataCollector
- PhpAb\Analytics\Google\UAnalyticsRenderer

Alternatively:

- PhpAb\Analytics\DataCollector\Google
- PhpAb\Analytics\Renderer\UAnalytics

Not sure which one to choose